js/interface.js

```diff
@@ -33,11 +33,15 @@ var dataSources;
 var trashedDataSources;
 var allDataSources;
 var table;
-var dataSourceEntriesHasChanged = false;
 var isShowingAll = false;
 var columns;
 var dataSourcesToSearch = [];
 var initialLoad = true;
+var entryMap = {
+  original: [],
+  entries: []
+};
+var locale = navigator.language.indexOf('en') === 0 ? navigator.language : 'en';
```

Comment:
What's this for?

Reply:
This is to make sure any date localisation uses an English locale that's closest to the browser language.